packages/design-system-docs/src/pages/components/TextField/LabelMask.example.jsx

```diff
@@ -0,0 +1,44 @@
+import { TextField } from '@design-system';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const Example = () => {
+  return (
+    <div>
+      <TextField
```

Comment:
We could pair down this example for now to be just for Date and do state management for a single input like
```jsx
const Example = () => {
  const [value, setValue] = useState("");
  return (
    <div>
      <TextField
        label="Enter the last day of your coverage"
        hint="If you don't have it, give your best estimate. For example: 01/02/2022"
        labelMask={LabelMask.DATE}
        value={value}

Reply:
We want a chance to showcase the simpler API anyway, where we want users to store the raw value from `onChange` without having to do masking or unmasking or anything.